pkg/pdapi/pdapi.go

```diff
@@ -315,7 +316,14 @@ func (c *pdClient) getStores(apiURL string) (*StoresInfo, error) {
 }
 
 func (c *pdClient) GetStores() (*StoresInfo, error) {
-	return c.getStores(fmt.Sprintf("%s/%s", c.url, storesPrefix))
+	storesInfo, err := c.getStores(fmt.Sprintf("%s/%s", c.url, storesPrefix))
+	if err != nil {
+		if strings.HasSuffix(err.Error(), tiKVNotBootstrapped+"\n") {
+			err = TiKVNotBootstrappedErrorf(tiKVNotBootstrapped)
```

Comment:
how about using `err.Error()` instead of `tiKVNotBootstrapped` here?

Reply:
But how can we know whether the error type is `NotBootstrapped`?

Reply:
`_, ok := err.(*TiKVNotBootstrappedError)` this should still work? I mean, only replace the error message with the original `err.Error()`.

Reply:
I see.